Formula/druid.rb

```diff
@@ -0,0 +1,35 @@
+class Druid < Formula
+  desc "High-performance, column-oriented, distributed data store"
+  homepage "http://druid.io"
+  url "http://static.druid.io/artifacts/releases/druid-0.9.2-bin.tar.gz"
+  sha256 "10cb45d36c72ba4e4b7029c35bdfae0974b7950241ab06acaa8f8e284d1a989f"
+
+  bottle :unneeded
+
+  depends_on "zookeeper"
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install Dir["*"]
+    cd libexec do
+      system "bin/init"
+    end
+  end
+
+  test do
+    ENV["DRUID_CONF_DIR"] = "conf-quickstart/druid"
+    ENV["DRUID_LOG_DIR"] = testpath
+    ENV["DRUID_PID_DIR"] = testpath
+
+    cd libexec do
```

Comment:
Yes, they would need to `cd`. We could have the script installed into `bin`, but there are a number of them.

```
broker.sh
coordinator.sh
historical.sh
jconsole.sh
middleManager.sh
node.sh
overlord.sh
```

Not sure what the Homebrew stance is on adding 7 scripts to `bin`. I assume we'd also want to prefix them with `druid-` as well.